doc/quickstart/install.rst

```diff
@@ -199,7 +199,49 @@ To install from source for development, follow these instructions.
 Pre-installed versions on HPC clusters
 --------------------------------------
 
+You will find the tool available on HPC clusters and there will be no need to install it
```

Comment:
This is nice documentation, but it appears to be not related to conda lock files. I think it would be better to move this to a separate pull request

Reply:
OK good point!

Reply:
dunnit https://github.com/ESMValGroup/ESMValCore/pull/1409